src/middleware/queries/settlements.ts

```diff
@@ -53,6 +54,45 @@ const instructionAttributes = `
 
 type InstructionArgs = 'id' | 'venueId' | 'status';
 
+/**
+ * Filters for instructions
+ *
+ */
+export interface InstructionPartiesFilters {
+  /**
+   * The DID of the identity to filter by
+   */
+  identity?: string;
```

Comment:
```suggestion
  identity?: (string | Identity);
```
Since we are using  the same interface as the method param, we need to support both. At all places where identity is there we add support for both